src/schema/v2/RenameFields.ts

```diff
@@ -89,7 +89,9 @@ export class RenameFields implements Transform {
       if (newName) {
         if (this.changedFields[fieldKey(type, newName)] !== undefined) {
           throw new Error(
-            `Cannot rename two fields to the same name: ${newName}`
+            `Cannot rename two fields (${
+              this.changedFields[fieldKey(type, newName)]
+            }, ${oldName}) to the same name ${newName} on type ${type.name}`
```

Comment:
Nice, did you find these because of this error?

Reply:
Yep! That error was helpful. I modified it so I could find the types easier though.